src/common/dataModels/records.ts

```diff
@@ -23,4 +23,4 @@ export interface IRecordExistsResponse {
   exists: boolean;
 }
 
-export interface IFindRecordResponse extends IRecordRequestParams, Partial<IRasterCatalogUpsertRequestBody> {}
+export interface IFindRecordResponse extends IRasterCatalogUpsertRequestBody {}
```

Comment:
why?

Reply:
'IRasterCatalogUpsertRequestBody' is now including "id" field so its unnecessary extends 'IRecordRequestParams'

Reply:
use Partial<IRasterCatalogUpsertRequestBody>